views/publication/preview.templ

```diff
@@ -363,7 +363,7 @@ templ Preview(c *ctx.Ctx, p *models.Publication, actions, downloadMainFileAction
 																				if a.Person != nil {
 																					<li class="c-meta-item">
 																						<i class="if if-ghent-university if--small if--muted"></i>
-																						<span class="text-muted">UGent author</span>
+																						<span class="text-muted">UGent</span>
```

Comment:
Cannot test this right now as none of the suggestions have users with an UGent ID.